plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -69,7 +69,7 @@
 
     {% if not showConfirmOnly %}
     {% set loadInNewWindow = isSafari and trackVisits %}
-    <form method="post" action="?module=CoreAdminHome&amp;action=optOut{% if language %}&amp;language={{ language }}{% endif %}{% if loadInNewWindow %}&amp;setCookieInNewWindow=1{% endif %}" {% if loadInNewWindow %}target="_blank"{% endif %}>
+    <form method="post" action="?{{ queryParameters }}" {% if loadInNewWindow %}target="_blank"{% endif %}>
```

Comment:
Even if query parameters were already URL encoded this (`{{ queryParameters }}`) would escape them again. I think either `{{ queryParameters|e('url') }}` and no `urlencode` in the `getQueryParameters` method should be used or `{{ queryParameters|raw }}` but not 100% sure